dockerfiles/buildall.sh

```diff
@@ -60,6 +60,13 @@ set -e
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 pushd $DIR 2>&1 > /dev/null
 
+. ../cmake_modules/parse_cmake.sh
+parse_cmake
+
+if [[ "$HPCC_MATURITY" = "release" ]] && [[ "$HPCC_MAJOR" = "7" ]] && [[ "$HPCC_MINOR" = "8" ]] ; then
```

Comment:
perhaps the major.minor latest should come from somewhere else, e.g. a ./latest file ?
so buildall.sh doesn't need to change when we want to up it.

Reply:
Need to change a file either way...

The concept is flawed though - tags on the old branch will still be compared against the old version, so will think they are still latest. Really we just want a boolean "is latest" that we set to false on the old branch at the same time we set to true on the new.